test/features/operator/install_explainability.feature

```diff
@@ -6,17 +6,55 @@ Feature: Kogito Explainability service
     Given Namespace is created
 
   @smoke
-  # Disabled as long as https://issues.redhat.com/browse/KOGITO-3176 is not solved
-  @disabled
   Scenario: Install Kogito Explainability
     Given Kogito Operator is deployed with Kafka operator
-    And Install Kogito Explainability with 1 replicas
+    # Workaround until https://issues.redhat.com/browse/KOGITO-3176 is solved
+    And Kafka topic "trusty-explainability-result" is deployed
+    When Install Kogito Explainability with 1 replicas
     Then Kogito Explainability has 1 pods running within 10 minutes
-    And HTTP GET request on service "explainability" is successful within 2 minutes with path "health/live"
 
 #####
 
   @events
+  @infinispan
+  @kafka
+  Scenario: Explainability retrieves explainability requests events
```

Comment:
I've created this new scenario to test the in-build integration with Kafka, Infinispan and the rest of the components.